process/peer/process.go

```diff
@@ -715,6 +738,28 @@ func (vs *validatorStatistics) getTempRating(s string) uint32 {
 	return peer.GetTempRating()
 }
 
+func (vs *validatorStatistics) updateRatingFromTempRating(pks []string) error {
+	rootHash, _ := vs.RootHash()
```

Comment:
err should be treated as not critical, but logged at least

Reply:
fixed